pkgs/development/libraries/nspr/4_26.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchurl
```

Comment:
Should we also call this file `…_latest` like we did with a few other packages? It is always tiresome to rename it on every bump.